packages/ts/hilla-frontend/src/types.d.ts

```diff
@@ -0,0 +1,23 @@
+import type { ConnectionIndicator, ConnectionStateStore } from '@vaadin/common-frontend';
+
+declare global {
+  interface VaadinRegistration {
+    is: string;
+    version?: string;
+  }
+
+  interface VaadinFlow {
+    clients?: Record<string, unknown>;
+  }
+
+  interface Vaadin {
+    Flow?: VaadinFlow;
+    connectionIndicator?: ConnectionIndicator;
+    connectionState?: ConnectionStateStore;
+    registrations?: VaadinRegistration[];
+  }
+
+  interface Window {
+    Vaadin?: Vaadin;
+  }
+}
```

Comment:
Can it clash with https://github.com/vaadin/sso-kit/blob/5c44f9c60a2c7799b47e66ae28b57d7324329711/sso-kit-client/react/src/index.ts#L18-L35 ?

Reply:
I was probably wrong on requesting changes: if needed, that's more on the kit side.

Reply:
It's ok 🙂 It's better to block merging to have an answer than to merge a wrong code.

> Can it clash
Well, it shouldn't, as far as I know. However, it's a good idea to make a separate check.

Reply:
Let's have it defined as a local interface